docs/developer/explanation/debian-package-publishing.rst

```diff
@@ -0,0 +1,104 @@
+Debian package publishing
```

Comment:
Nit: The page title says "Debian package publishing" (and yes, Ubuntu packages are .deb packages) and the document covers only the process for the Ubuntu primary archive and not the PPAs. This feels a little bit confusing to me. So I suggest renaming the page title to `Ubuntu package publishing` and clarify that this document only covers the details of the Ubuntu upload and publishing process.